librz/core/cmd_descs/cmd_info.yaml

```diff
@@ -436,8 +436,9 @@ commands:
       - RZ_OUTPUT_MODE_JSON
     args: []
   - name: iz
-    cname: cmd_info_strings(see also str.search.*)
+    cname: cmd_info_strings
     summary: List strings
+    description: see also str.search.*
```

Comment:
Be a bit more descriptive: `Lists the strings found in a binary; the behavior can be modified via str.search.*`